pkg/metrics/metrics.go

```diff
@@ -1105,6 +1110,17 @@ func CreateConfiguration(metricsEnabled []string) (Configuration, []prometheus.C
 			collectors = append(collectors, KubernetesCNPStatusCompletion)
 			c.KubernetesCNPStatusCompletionEnabled = true
 
+		case Namespace + "_terminating_endpoints_events_total":
+			TerminatingEndpointsEvents = prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
Does having nodename as label provide any value ? Just my thinking about having one more dimension for troubleshooting (e.g. problematic node).

Reply:
Sure, good idea! Thanks!